lib/nerves_metal_detector/inventory/data/product_update_items/chicago_electronic_distributors_us.ex

```diff
@@ -0,0 +1,28 @@
+defmodule NervesMetalDetector.Inventory.Data.ProductUpdateItems.ChicagoElectronicDistributorsUs do
+  alias NervesMetalDetector.Vendors.ChicagoElectronicDistributorsUs.ProductUpdate
+
+  @items [
+    # RPi 4
+    %ProductUpdate{
+      url: "https://chicagodist.com/products/raspberry-pi-4-model-b-2gb",
+      sku: "RPI4-MODBP-2GB"
+    },
+    %ProductUpdate{
+      url: "https://chicagodist.com/collections/raspberry-pi/products/raspberry-pi-4-model-b-1gb",
```

Comment:
Lets change the url to `https://chicagodist.com/products/raspberry-pi-4-model-b-1gb` to be consistent with the other urls.